src/views/v2/list/event/title.php

```diff
@@ -16,7 +16,7 @@
  * @see tribe_get_event() For the format of the event object.
  */
```

Comment:
:no_entry_sign: There must be exactly one blank line after the file comment

<sub>hash: 8ccca0725768a653915374b123fe71c5</sub>